crates/ibc/src/relayer/ics18_relayer/utils.rs

```diff
@@ -111,14 +111,6 @@ mod tests {
             // Update client on chain B to latest height of A.
             // - create the client update message with the latest header from A
             let a_latest_header = ctx_a.query_latest_header().unwrap();
-            assert_eq!(
```

Comment:
I think we should get rid of the entire `crates/ibc/relayer/` directory. `RelayerContext` is only implemented on `MockContext`; we should move the used methods there directly. Otherwise the test in `utils.rs` should be moved somewhere in the `ics26_routing` directory.

Reply:
I removed the `relayer` and moved `ics18_relayer` under the mock module. https://github.com/cosmos/ibc-rs/pull/314/commits/65af6b50807ee701344777210816ffb1fe4432e3
Do you think it makes sense?

Reply:
Good idea yes. While we're at it, I think we should:

- Remove `utils.rs` and move the contents in `context.rs` (doesn't make sense to me why this big test is a `utils` module
- move `build_client_update_datagram()` in the `tests` submodule, since it's really a helper function for the test

WDYT?

Reply:
Agree, Will do so